src/pal/pal_concept.h

```diff
@@ -91,6 +97,7 @@ namespace snmalloc
     ConceptPAL_static_members<PAL> &&
     ConceptPAL_error<PAL> &&
     ConceptPAL_memops<PAL> &&
+    ConceptPAL_get_entropy64<PAL> &&
```

Comment:
Please gate this on the `Entropy` `PalFeatures` flag:

```suggestion
    (!pal_supports<Entropy, PAL> ||
      ConceptPAL_get_entropy64<PAL>) &&
```